jax/lax/__init__.py

```diff
@@ -336,3 +336,6 @@
   psum_p,
   pswapaxes,
 )
+from .lax_other import (
+  conv_general_dilated_patches
```

Comment:
To get this documented, also add this function to the list in `jax.lax.rst`.

Reply:
Thanks, added!